colossus/src/main/scala/colossus/protocols/http/HttpBody.scala

```diff
@@ -27,9 +29,7 @@ class HttpBody(private val body: Array[Byte], val contentType : Option[HttpHeade
 
   override def toString = bytes.utf8String
 
-  def withContentType(contentType: String) = withContentTypeHeader(HttpHeader("Content-Type", contentType))
-
-  def withContentTypeHeader(header: HttpHeader) = new HttpBody(body, Some(header))
```

Comment:
Again, we shouldn't be making API changes in a minor release.  Also in general I don't think these should be removed.  Just because nothing _inside_ the framework uses this method doesn't necessarily mean that nobody _using_ the framework is using it.